Tasks02_05/task03.py

```diff
@@ -13,8 +13,27 @@
         ...
 
 """
+import math
 from typing import Tuple
 
 
 def find_maximum_and_minimum(file_name: str) -> Tuple[int, int]:
-    ...
+    """
+    read a file line by line that consists of str
+    str include numbers (separated with commas)
+    :param file_name: name of file + .txt
+    :return: Tuple(min_value, max_value) for each line
+    """
+    with open(file_name) as fi:
+        for line in fi:
+            line = line.split(",")
```

Comment:
line = line.split(",")

line on the left is not actually line after assignment, it's list of numbers. try to use another name for the variable